### PR TITLE
Anya/836 download files estimate

### PR DESCRIPTION
**Pull request by aroltsch**
connects #836

### CODE REVIEW

File: app/models/record.rb [hunk 1]
```diff
@@ -29,6 +29,8 @@ class Record < ActiveRecord::Base
   delegate :manifest, :service, to: :manifest_source
   delegate :file_number, to: :manifest
 
+  AVERAGE_DOWNLOAD_TIME = 2
```
Comment (mdbenjam) [inline app/models/record.rb]: nit: could we add _IN_SECONDS to this const? Just makes it super clear.

File: app/models/serializers/v2/files_download_serializer.rb [hunk 2]
```diff
@@ -5,6 +5,7 @@ class Serializers::V2::FilesDownloadSerializer < ActiveModel::Serializer
   attribute :fetched_files_at
   attribute :number_successful_documents
   attribute :number_failed_documents
+  attribute :time_to_complete
```
Comment (mdbenjam) [inline app/models/serializers/v2/files_download_serializer.rb]: Do we need to send seconds left as well? How will we draw the progress bar?

Comment (aroltsch) [reply app/models/serializers/v2/files_download_serializer.rb]: Ah, great point!

File: app/models/record.rb (hunk 1 above)
Comment (aroltsch) [reply app/models/record.rb]: good idea!

**Pull request merged**